restler/restler_settings.py

```diff
@@ -512,11 +514,23 @@ def disable_logging(self):
 
     @property
     def client_certificate_path(self):
-        return self._client_certificate_path.val
+        if self._client_certificate_path:
+            return self._client_certificate_path.val
+        else:
+            try:
```

Comment:
> try: [](http://example.com/codeflow?start=11&length=5)

Please switch this and the other occurrences to the pattern that checks for inclusion and remove the exception handling, i.e.:
```
if 'certificate' in self._authenticationsettings.val:
   cert=self._auth....val['certificate']
   if 'cert_location' in cert:
     ...
   else
      error?
```

Reply:
updated, thanks, please take a look

Reply:
will break these over multiple lines as suggested